Sources/NIOCore/EventLoop.swift

```diff
@@ -13,7 +13,12 @@
 //===----------------------------------------------------------------------===//
 
 import NIOConcurrencyHelpers
+#if canImport(Dispatch)
 import Dispatch
+#elseif canImport(WASILibc)
```

Comment:
The `elseif` implies a relationship between these two ideas, but I think they're fundamentally unrelated.